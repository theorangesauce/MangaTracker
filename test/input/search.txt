--SEARCH TEST----
S
Author 1
S
Author 2
S
Author 3
S
Publisher 1
S
Publisher 2
S
Alternate Name
S
Alternate Title
X
--END SEARCH-----
