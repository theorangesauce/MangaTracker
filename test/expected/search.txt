Author 1
  Series 1
  Series 2

Author 2
  Series 3
  Series 5

Author 3
  Last Series
  Series 4

Publisher 1
  Series 1
  Series 4
  Series 5

Publisher 2
  Last Series
  Series 2
  Series 3

Alternate Name
  Last Series
  Series 1

Alternate Title
  Series 2
  Series 3
  Series 4
  Series 5
